pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/Participant.java

```diff
@@ -43,97 +43,66 @@ public class Participant implements Cloneable {
             + "LEFT JOIN ddp_onc_history o on (o.participant_id = p.participant_id) "
             + "LEFT JOIN ddp_participant_record r on (r.participant_id = p.participant_id) "
             + "LEFT JOIN ddp_participant_exit ex on (p.ddp_participant_id = ex.ddp_participant_id "
-            + "AND p.ddp_instance_id = ex.ddp_instance_id) "
-            + "WHERE realm.instance_name = ? ";
+            + "AND p.ddp_instance_id = ex.ddp_instance_id) " + "WHERE realm.instance_name = ? ";
```

Comment:
```suggestion
            + "AND p.ddp_instance_id = ex.ddp_instance_id) WHERE realm.instance_name = ? ";
```
since you consolidated lines, might as well remove the extra concatentation

Reply:
was auto-consolidated by stupidJ. I will leave it as it is for now, because otherwise I need someone to approve it again, but I will try to keep it in mind and change it with the next PR